src/main/java/persistence/entity/EntityManagerImpl.java

```diff
@@ -0,0 +1,37 @@
+package persistence.entity;
+
+import jdbc.JdbcTemplate;
+import jdbc.RowMapperImpl;
+import persistence.sql.dml.builder.DeleteQueryBuilder;
+import persistence.sql.dml.builder.InsertQueryBuilder;
+import persistence.sql.dml.builder.SelectQueryBuilder;
+
+public class EntityManagerImpl<T> implements EntityManager<T> {
+
+    private final JdbcTemplate jdbcTemplate;
+    private final SelectQueryBuilder selectQueryBuilder;
+    private final InsertQueryBuilder insertQueryBuilder;
+    private final DeleteQueryBuilder deleteQueryBuilder;
+
+    public EntityManagerImpl(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+        this.selectQueryBuilder = new SelectQueryBuilder();
+        this.insertQueryBuilder = new InsertQueryBuilder();
+        this.deleteQueryBuilder = new DeleteQueryBuilder();
+    }
+
+    @Override
+    public T find(Class<T> clazz, Long id) {
+        return jdbcTemplate.queryForObject(selectQueryBuilder.findAll(clazz), new RowMapperImpl<>(clazz));
+    }
```

Comment:
```suggestion
    public T find(Class<T> clazz, Long id) {
        return jdbcTemplate.queryForObject(selectQueryBuilder.findById(clazz, id), new RowMapperImpl<>(clazz));
    }
```
메소드 호출 실수가 있으셨던것 같네요 🙂
findAll 까지 고려해 주신 부분은 대단하세요 🥇

Reply:
앗 그렇네요:) 체크 감사합니다! 
2주차 0단계에서 적용해야겠군요 ㅎㅎ